bin/cache.ml

```diff
@@ -21,33 +21,44 @@ let trim =
     Cmd.info "trim" ~doc ~man
   in
   Cmd.v info
-  @@ let+ trimmed_size =
-       Arg.(
-         value
-         & opt (some bytes) None
-         & info ~docv:"BYTES" [ "trimmed-size" ] ~doc:"Size to trim from the cache.")
-     and+ size =
-       Arg.(
-         value
-         & opt (some bytes) None
-         & info ~docv:"BYTES" [ "size" ] ~doc:"Size to trim the cache to.")
-     in
-     Log.init_disabled ();
-     let open Result.O in
-     match
-       let+ goal =
-         match trimmed_size, size with
-         | Some trimmed_size, None -> Result.Ok trimmed_size
-         | None, Some size ->
-           Result.Ok (Int64.sub (Dune_cache.Trimmer.overhead_size ()) size)
-         | _ -> Result.Error "please specify either --size or --trimmed-size"
-       in
-       Dune_cache.Trimmer.trim ~goal
-     with
-     | Error s -> User_error.raise [ Pp.text s ]
-     | Ok { trimmed_bytes } ->
-       User_message.print
-         (User_message.make [ Pp.textf "Freed %s" (Bytes_unit.pp trimmed_bytes) ])
+  @@
```

Comment:
@snowleopard It seems there is noise anyway due to ocamlformat. Just know that I've only touched the documentation for `size` and `trimmed_size` in the following block.

Reply:
If you put `@@` and `let+` on the same line and reformat, it will use that style and that will decrease the diff.

Reply:
Thanks! I did not know that.